pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -1580,7 +1580,22 @@ func newVMImportForCR(cr *hcov1beta1.HyperConverged, namespace string) *vmimport
 
 	spec := vmimportv1beta1.VMImportConfigSpec{}
 	if cr.Spec.Infra.NodePlacement != nil {
-		cr.Spec.Infra.NodePlacement.DeepCopyInto(&spec.Infra)
+		if cr.Spec.Infra.NodePlacement.Affinity != nil {
+			affinity := corev1.Affinity{}
+			cr.Spec.Infra.NodePlacement.Affinity.DeepCopyInto(&affinity)
+			spec.Infra.Affinity = &affinity
+		}
+		if cr.Spec.Infra.NodePlacement.NodeSelector != nil {
+			spec.Infra.NodeSelector = make(map[string]string)
+			for k, v := range cr.Spec.Infra.NodePlacement.NodeSelector {
+				spec.Infra.NodeSelector[k] = v
+			}
+		}
+		for _, hcoTolr := range cr.Spec.Infra.NodePlacement.Tolerations {
+			tvTolr := corev1.Toleration{}
+			hcoTolr.DeepCopyInto(&tvTolr)
+			spec.Infra.Tolerations = append(spec.Infra.Tolerations, tvTolr)
+		}
```

Comment:
@pkliczewski - Are you sure this is needed? Deepcopy for node placement worked very well in unit tests. sdkapi.NodePlacement implements `DeepCopyInto`. Also, we want the operator to use sdkapi.NodePlacement, and to avoid this component specific migrations.

Reply:
Yes, you can check it yourself. Compiler complains. We are using newer version of controller sdk so I had to stop using `DeepCopyInto`

Reply:
OK, I found it VM-Import moved from `github.com/kubevirt/controller-lifecycle-operator-sdk` to `kubevirt.io/controller-lifecycle-operator-sdk`. This is why HCO does not consider it the same version.

I changed it in #859. I will close this PR.